network-api/networkapi/wagtailpages/models.py

```diff
@@ -15,6 +15,8 @@
 from wagtail.core.models import Orderable as WagtailOrderable
 from wagtail.images.models import Image
 from modelcluster.fields import ParentalKey
+from modelcluster.contrib.taggit import ClusterTaggableManager
```

Comment:
tag related code is basically copied from Wagtail doc: http://docs.wagtail.io/en/v2.0/reference/pages/model_recipes.html#tagging